bower.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "paper-tristate-checkbox",
-  "version": "0.1.0",
+  "version": "0.3.0",
```

Comment:
Change this to 0.2.0, when we release we'll do a version bump on this file as well.